### ISSUE TITLE
Availability of SQL management SDK 

### ISSUE THREAD

**Issue by cgwrench** [Azure/azure-sdk-for-net#18017]
I wanted to try out the SQL management SDK. The [README](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/sqlmanagement/Azure.ResourceManager.Sql/README.md) states that this is available on NuGet. However, this doesn't seem to be the case. Should this be available on NuGet? If not, when can we expect to see a preview release of this SDK?

**Comment by jsquire** [comment]
Thank you for your feedback.  Tagging and routing to the team member best able to assist.

**Comment by allenjzhang** [comment]
New SQL Management SDK work started late in 2020 but has been put on hold as we are refining the new management API surfaces. Once that work is done, we will be refreshing the currently released preview Azure.ResourceManagment.* SDKs. SQL would be updated and released at that time. The anticipated timeline is in a few months. Please stay tuned.
